sdk/storage/azblob/README.md

```diff
@@ -0,0 +1,224 @@
+# Azure Storage Blob SDK for Go
+
+## Introduction
+
+The Microsoft Azure Storage SDK for Go allows you to build applications that takes advantage of Azure's scalable cloud
+storage. This SDK replaces the previously previewed [azblob package](https://github.com/azure/azure-storage-blob-go) in the separate repository.
+
+## Prerequisites
+
+* Go versions 1.16 or higher. If you don't already have it, install [the Go distribution](https://golang.org/dl/)
+* You must have an [Azure storage account][azure_storage_account]
+
+## Getting Started
+
+* Install the Azure blob storage client library for Go with `go get`:
+  ```bash
+  go get github.com/Azure/azure-sdk-for-go/sdk/storage/azblob
+  ```
+* Import SDK in your code:
+  ```golang
+  import "github.com/Azure/azure-sdk-for-go/sdk/storage/azblob"
+  ```
+
+## Code Samples
+
+`azblob` allows you to interact with three types of resources
+
+* The storage accounts.
+* The containers within those storage accounts.
+* The blobs (block blob/ page blob/ append blob) within those containers.
+
+To interact with these resources, start by creating the instances of each type
+
+### Types of credentials
+
+The clients support different forms of authentication. Cosmos accounts can use a Shared Key Credential, Connection
+String, or an Shared Access Signature Token for authentication. Storage account can use the same credentials as a Cosmos
+account and can use the credentials
+in [`azidentity`](https://pkg.go.dev/github.com/Azure/azure-sdk-for-go/sdk/azidentity)
+like `azidentity.NewDefaultAzureCredential()`.
+
+The azblob package supports any of the types that implement the `azcore.TokenCredential` interface, authorization via a
+Connection String, or authorization with a Shared Access Signature Token.
+
+#### 1. Creating the client from a shared key
+
+To use an account [shared key][azure_shared_key] (aka account key or access key), provide the key as a string. This can
+be found in your storage account in the [Azure Portal][azure_portal_account_url] under the "Access Keys" section or by
+running the following Azure CLI command:
+
+```bash
+az storage account keys list -g MyResourceGroup -n MyStorageAccount
+```
+
+Use Azure Active Directory (AAD) authentication as the credential parameter to authenticate the client:
+
+```golang
+cred, err := azidentity.NewDefaultAzureCredential(nil)
+handle(err)
+serviceClient, err := azblob.NewServiceClient("https://<myAccountName>.blob.core.windows.net/", cred, nil)
+handle(err)
+```
+
+##### 2. Creating the client from a connection string
+
+Depending on your use case and authorization method, you may prefer to initialize a client instance with a connection
+string instead of providing the account URL and credential separately. To do this, pass the connection string to the
```

Comment:
I think you can remove this first sentence and touch up the second a bit. Go prefers brevity over verbosity so we try to reflect that in docs as well.